[2020-12-17 22:12:59,022] WARN Client session timed out, have not heard from server in 12388ms for sessionid 0x10000828e8a0000 (org.apache.zookeeper.ClientCnxn)
[2020-12-17 22:12:59,320] INFO Client session timed out, have not heard from server in 12388ms for sessionid 0x10000828e8a0000, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-12-17 22:13:00,026] INFO [GroupCoordinator 0]: Member consumer-console-consumer-16584-1-e18b4ed7-59de-4a72-a713-a1aa92680d85 in group console-consumer-16584 has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2020-12-17 22:13:00,034] INFO [GroupCoordinator 0]: Preparing to rebalance group console-consumer-16584 in state PreparingRebalance with old generation 1 (__consumer_offsets-15) (reason: removing member consumer-console-consumer-16584-1-e18b4ed7-59de-4a72-a713-a1aa92680d85 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2020-12-17 22:13:00,078] INFO [GroupCoordinator 0]: Group console-consumer-16584 with generation 2 is now empty (__consumer_offsets-15) (kafka.coordinator.group.GroupCoordinator)
[2020-12-17 22:13:00,269] INFO [GroupCoordinator 0]: Preparing to rebalance group console-consumer-16584 in state PreparingRebalance with old generation 2 (__consumer_offsets-15) (reason: Adding new member consumer-console-consumer-16584-1-18f2add6-f7e5-473a-81f5-8486b26361df with group instance id None) (kafka.coordinator.group.GroupCoordinator)
[2020-12-17 22:13:00,289] INFO [GroupCoordinator 0]: Stabilized group console-consumer-16584 generation 3 (__consumer_offsets-15) (kafka.coordinator.group.GroupCoordinator)
[2020-12-17 22:13:00,318] INFO [GroupCoordinator 0]: Assignment received from leader for group console-consumer-16584 for generation 3 (kafka.coordinator.group.GroupCoordinator)
[2020-12-17 22:13:01,752] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-12-17 22:13:01,782] INFO Socket connection established, initiating session, client: /127.0.0.1:53914, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2020-12-17 22:13:02,026] INFO Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x10000828e8a0000, negotiated timeout = 18000 (org.apache.zookeeper.ClientCnxn)
[2020-12-17 22:15:17,108] INFO Terminating process due to signal SIGHUP (org.apache.kafka.common.utils.LoggingSignalHandler)
[2020-12-17 22:15:17,111] INFO Terminating process due to signal SIGHUP (org.apache.kafka.common.utils.LoggingSignalHandler)
[2020-12-17 22:15:17,464] INFO [KafkaServer id=0] shutting down (kafka.server.KafkaServer)
[2020-12-17 22:15:17,514] INFO [KafkaServer id=0] Starting controlled shutdown (kafka.server.KafkaServer)
[2020-12-17 22:15:17,956] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection reset (org.apache.zookeeper.ClientCnxn)
[2020-12-17 22:15:18,122] INFO [ZooKeeperClient Kafka server] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2020-12-17 22:15:18,122] INFO [ZooKeeperClient Kafka server] Connected. (kafka.zookeeper.ZooKeeperClient)
[2020-12-17 22:15:18,150] INFO [GroupCoordinator 0]: Member[group.instance.id None, member.id consumer-console-consumer-16584-1-18f2add6-f7e5-473a-81f5-8486b26361df] in group console-consumer-16584 has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2020-12-17 22:15:18,150] INFO [GroupCoordinator 0]: Preparing to rebalance group console-consumer-16584 in state PreparingRebalance with old generation 3 (__consumer_offsets-15) (reason: removing member consumer-console-consumer-16584-1-18f2add6-f7e5-473a-81f5-8486b26361df on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2020-12-17 22:15:18,151] INFO [GroupCoordinator 0]: Group console-consumer-16584 with generation 4 is now empty (__consumer_offsets-15) (kafka.coordinator.group.GroupCoordinator)
[2020-12-17 22:15:19,838] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-12-17 22:15:19,849] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2020-12-17 22:15:19,953] INFO [ZooKeeperClient Kafka server] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
